lib/factor/factorQuadratic.js

```diff
@@ -262,8 +290,17 @@ function factorSumProductRule(node, symbol, aValue, bValue, cValue, negate) {
         const secondParen = Node.Creator.parenthesis(
           Node.Creator.operator('+', [ux, v]));
 
+        // SUBSTEP: ux(rx + s) + v(rx + s)
```

Comment:
can you do a `before -> after` here too like you did above? :D

(and add it below too)